charts/opensearch-dashboards/templates/service.yaml

```diff
@@ -13,6 +13,9 @@ metadata:
 {{- end }}
 spec:
   type: {{ .Values.service.type }}
+  {{- if (semverCompare ">= 1.23-0" .Capabilities.KubeVersion.Version) }}
+  ipFamilyPolicy: {{ .Values.service.ipFamilyPolicy }}
```

Comment:
Hey @UXabre thanks for PR, this is good change to have, can you please let us know what would be the default value, is this a breaking change if a chart is updated with `helm upgrade` ?
Thank you

Reply:
Hi, I actually set the default value to SingleStack to be more compatible (as many people don't use dual stack yet). However, now I'm thinking about it, I think it's safer to just check if the variable is set and thus fallback to the default that the CNI provides (at least, to my understanding the CNI would define the implicit default).

Also, another change, to be more versatile would be to also provide a config for the IPFamilies as well, this way, one could set it to singlestack and ipv6 (if they are brave :p).

Curious to hear what you think of this proposal.